api/src/services/server.ts

```diff
@@ -59,6 +60,14 @@ export class ServerService {
 			} else {
 				info.rateLimit = false;
 			}
+			if (env.RATE_LIMITER_GLOBAL_ENABLED) {
+				info.rateLimitGlobal = {
+					points: env.RATE_LIMITER_GLOBAL_POINTS,
+					duration: env.RATE_LIMITER_GLOBAL_DURATION,
+				};
+			} else {
+				info.rateLimit = false;
```

Comment:
Could this be meant to be `rateLimitGlobal` instead of `rateLimit`?

Reply:
That is indeed what it is supposed to be 😬thats what i get for doing copy-paste.